examples/faces_detection/faces_detection.ipynb

```diff
@@ -0,0 +1,987 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !pip install toloka-kit # To interact with Toloka API
</pre><p>Please. specify the version of toloka-kit (1.0.0 is current version)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Toloka/toloka-kit/pull/119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>